src/Components/ProvisioningWizard/steps/FinishProgress/index.js

```diff
@@ -117,31 +121,26 @@ const FinishStep = ({ onClose, imageID }) => {
     : false;
```

Comment:
Do we have to use the ternary operator? Would possibly `polledReservation && polledReservation.success === false` not work? I've been only working in typescript if that makes a difference.

Reply:
maybe even using the conditional operator `polledReservation?.success === false` would work here :)
It's already existing code, do you want to send a patch, or shall we include it here? :)

Reply:
Let's include this here since you already modify the file in this PR. :)